docs/v1/P4Runtime-Spec.mdk

```diff
@@ -6284,5 +6284,21 @@ clients are aware that the server is using a larger maximum receive message
 size. The gRPC server running the P4Runtime service must not set the maximum
 receive message size to a value smaller than the default (4MB).
 
+### gRPC Default Port
+
+IANA has allocated TCP port number 9559 for the P4Runtime service, and
+P4Runtime server implementations should adopt port 9559 as the default. Users
```

Comment:
maybe we could use a stronger verb here, s/should/must/?

Reply:
updated to "must"